### ISSUE TITLE
Zen-coding 0.7 for Dreamweaver CSS selector expand bug

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#232]
```
What steps will reproduce the problem?

1. Dreamweaver CS5 + zen-coding 0.7
2. Open CSS file
3. Try to expand attribute that placed right after opening curly brace "{"
For example:

"body {m"->tab will expand it to:

"<body {m></body {m>"

Seems, its not recognize CSS file as CSS mode (and not HTML)
```

---

Original issue reported on code.google.com by `sublimey...@gmail.com` on 23 Dec 2010 at 2:40

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `GreLIm...@gmail.com` on 24 Dec 2010 at 2:17
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 0.7.1
```

---

Original comment by `GreLIm...@gmail.com` on 10 Jan 2011 at 12:09
- Changed state: **Fixed**
